scripts/ci/30_test.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 set -euxo pipefail
 
-SERVERSIDE_FEATURES_ON=\"\" yarn run test "${APP}" --runInBand
\ No newline at end of file
+EXTRA_OPTS=""
+
+projects_uncollectible_coverage=("contentful-translation-extension" "application-templates-no-debt-certificate" "api-domains-email-signup" "skilavottord-web" "shared-babel")
```

Comment:
These projects refused to provide code coverage information and generated an error when tried to do so. I attempted downgrading and upgrading `jest` & co. but did not help. Leaving those as a future task hoping that upgrades in the future might reveal what the issue is.